debugging/coreclr/dockerDebugConfigurationProvider.ts

```diff
@@ -71,7 +71,9 @@ export class DockerDebugConfigurationProvider implements DebugConfigurationProvi
     }
 
     public resolveDebugConfiguration(folder: WorkspaceFolder | undefined, debugConfiguration: DockerDebugConfiguration, token?: CancellationToken): ProviderResult<DebugConfiguration | undefined> {
-        return this.resolveDockerDebugConfiguration(folder, debugConfiguration);
+        return callWithTelemetryAndErrorHandling(
+            'debugCoreClr',
+            async () => await this.resolveDockerDebugConfiguration(folder, debugConfiguration));
```

Comment:
nit: debugCoreClr doesn't seem to say what it's really doing. How about "resolveDebugConfiguration"?

Reply:
I chose `debugCoreClr` as that was *effectively* what was being done (i.e. an attempt to initiate debugging of .NET Core in Docker) even though `resolveDebugConfiguration` is what VS Code happens to name the interface function (and PMs looking at the telemetry data typically care more about the intent than the implementation specifics).  I'm amenable to other IDs or schemes; are there any conventions/restrictions in the naming (as the codebase uses a couple different styles)?

Reply:
If that's what it's effectively doing, it's fine, thanks.